### PR TITLE
doc: Remove apl gsg and merge contents into rt gsg

### PR DESCRIPTION
**Pull request by lirui34**
Change APL getting started guide to KBL getting started guide;
Merge some of the contents into preempt-rt getting started guide;
Move the modified kbl getting started guide to the behind of the configuration tutorials;
Fix issues for acrn_quick_setup.sh script.

### CODE REVIEW

**Comment by acrnsi** [comment]
OK to verify

**Comment by NanlinXie** [comment]
@deb-intel , Please help to check and merge this PR, thanks!

File: doc/faq.rst [hunk 1]
```diff
@@ -43,44 +45,53 @@ details:
...
-
-  config HV_RAM_SIZE
-    hex "Size of the RAM region used by the hypervisor"
-    default 0x07800000 if PLATFORM_SBL
-    default 0x0f000000 if PLATFORM_UEFI
+For example, if the NUC's physical memory size is 32G, you may follow these steps
+to make the new uefi ACRN hypervisor, and then deploy it onto the NUC board to boot
+ACRN Service VM with the 32G memory size.
+
+#. Modify the ``hypervisor/arch/x86/Kconfig`` from the ACRN source on host machine.
```
Comment (gvancuts) [inline doc/faq.rst]: We should really avoid having people modify the `Kconfig` file directly. This should instead be done using `make menuconfig`

File: doc/getting-started/rt_industry.rst [hunk 2]
```diff
@@ -6,9 +6,9 @@ Getting Started Guide for ACRN Industry Scenario
 Verified version
 ****************
 
-- Clear Linux version: **31080**
-- ACRN-hypervisor tag: **v1.3**
-- ACRN-Kernel(Service VM kernel): **acrn-2019w39.1-140000p**
+- Clear Linux version: **31470**
+- ACRN-hypervisor tag: **v1.4**
+- ACRN-Kernel(Service VM kernel): **4.19.78-95.iot-lts2018-sos**
 - ACRN-Kernel(Preempt-RT kernel): **acrn-2019w39.1-143000p**
```
Comment (gvancuts) [inline doc/getting-started/rt_industry.rst]: No need to update the version for the Preempt-RT kernel?

**Pull request merged**